clearwater-snmp-alarm-agent.root/usr/share/clearwater/bin/clearwater-snmp-alarm-agent-wrapper

```diff
@@ -47,7 +48,7 @@ get_daemon_args()
   # Don't try and load MIBS at startup - we don't need them and this just causes lots of
   # "missing MIB" error logs.
   export MIBS=""
-  DAEMON_ARGS="--snmp-ips $snmp_ip --local_ip $local_ip --community $snmp_community --log-dir $log_directory --log-level $log_level"
+  DAEMON_ARGS="--snmp-ips $snmp_ip --local_ip $local_ip --community $snmp_community --snmp_notification_type $snmp_notification_type --log-dir $log_directory --log-level $log_level"
```

Comment:
Have you tested what happens if a user sets snmp_notification_type= in the config file?